lib/NetPacksBase.h

```diff
@@ -34,8 +34,8 @@ struct DLL_LINKAGE CPack
 {
 	std::shared_ptr<CConnection> c; // Pointer to connection that pack received from
 
-	CPack() : c(nullptr) {};
-	virtual ~CPack() {};
+	CPack() : c(nullptr) {}

```

Comment:
Non-default constructor is unnecessary since shared_ptr is always initialized to nullptr

Reply:
Constructor is required here, but I can change it to default if desired.

Will post next series of modernization this evening)

Reply:
done